components/ListItem.js

```diff
@@ -53,11 +54,17 @@ export default class ListItem extends Component {
                         inputValue={this.state.inputValue}
                         onInputChange={this.updateInputValue}
                         buttonText='update' />
-                    :
+                :
+                (this.props.deleteItem !== null) && (this.props.updateItem !== null) ?
```

Comment:
y akie puede simplemente verificar q this.props.deleteItem && this.props.updateItem